h/templates/groups/share.html.jinja2

```diff
@@ -12,19 +12,24 @@
   <div class="content content--narrow">
     <div class="group-form is-member-of-group">
       <i class="h-icon-group group-form__heading-icon"></i>
-      <div class="group-form__heading">{{ group.name }}</div>
+      <div class="group-form__heading-short">{{ group.name }}</div>
 
       {% if document_links %}
-        Recently annotated documents:
-        <ul class="group-form-document-list">
+        <a href="//hypothes.is/stream?q=group:{{ group.pubid }}"
```

Comment:
This URL will point to the wrong host when using a development server. You can get the correct host using `{{ request.route_url('stream') }}?q=group:{{ group.pubid }}`